python/pyiceberg/table/__init__.py

```diff
@@ -69,21 +71,268 @@
     import ray
     from duckdb import DuckDBPyConnection
 
+    from pyiceberg.catalog import Catalog
 
 ALWAYS_TRUE = AlwaysTrue()
 
 
+class AlterTable:
```

Comment:
This looks more like a Transaction to me. Is there a better name?

Reply:
`TableUpdates`?